pyoos/parsers/ioos/one/describe_sensor.py

```diff
@@ -67,7 +74,7 @@ def __init__(self, element):
 
         self.shortName = self.get_ioos_def('shortName', 'identifier', ont)
         self.longName = self.get_ioos_def('longName', 'identifier', ont)
-        self.keywords = map(unicode, self.system.keywords)
+        self.keywords = list(map(str, self.system.keywords))
```

Comment:
Good question. Do we have `unicode` keywords? Should it be `string_types` instead? I am inclined to the latter.